### ISSUE TITLE
Accessibility - unable to tab through modals

### ISSUE THREAD

**Issue by joeyre** [kylefox/jquery-modal#268]
When tabbing through the page, I can hit enter to open a modal, but the tab focus remains in the background so I'm unable to tab through the contents of the modal

**Comment by ovrzls** [comment]
I would love to see this addressed. I have tried many ways to focus on the modal and have not been able to. Accessibility is a primary issue for me now that the site is launched. I need to change focus after the modal fades up (also tried no fade) inorder for keyboard access.

`

jQuery(document).ready( function($) {

	$('.modal').on('shown.bs.modal', function() {
		console.log('modal launched');
	});//nothing...

	$('.modal').on('show.bs.modal', function() {
		console.log('modal launched');
	});//nope...

	$('#id-for-modal')
		.on('show.bs.modal', function() {
		alert('shown!');
	}).modal('show');//modal shows but no alert...

	$(document).on('show.bs.modal','.modal', function () {
		alert('hi');
	});//yep still nothing

});

`

**Comment by Zenoo** [comment]
#294 Should fix the issue.
